hideShowPassword.js

```diff
@@ -52,6 +52,12 @@
     // set after input has been inserted into the DOM.
     enable: canSetInputAttribute,
 
+    // Set to true to focus the cursor in the input element after
+    // the element has been toggled. Benefitial for those who are
+    // typing out a password and just want to check their value
+    // and retain the focus of the input.
+    focusAfterToggle: false,
```

Comment:
@tylersticka I updated the PR using the `triggerOnToggle` syntax like you recommended to maintain consistency.